perception/lidar_centerpoint/lib/include/nms_kernel.hpp

```diff
@@ -0,0 +1,32 @@
+// Copyright 2022 TIER IV, Inc.
```

Comment:
[nits] rename to `circle_nms_kernel.hpp`

Reply:
changed at [b108e57](https://github.com/autowarefoundation/autoware.universe/pull/480/commits/b108e5746ba83aa2b01241b16dd99aca1edda90c)